### DISCUSSION TITLE
Porting to Filamentphp

### DISCUSSION THREAD

**Discussion by meeftah** [joshhanley/livewire-autocomplete#27] [Ideas]
I want to porting this to Filamentphp and want to ask your permission, is it okay?

**Comment by joshhanley** [comment]
@meeftah first off, thanks for asking, it's much appreciated 🙂

Does Filament not have any combobox/ autocomplete component already?

At this stage, I would probably say hold off on it, as I'm currently working on a new version that works better with Livewire V3, as such will also work better with Filament V3. It has a better API which makes it easier to use but offers more flexibility.

**Reply by meeftah** [reply]
you are welcome...as far as I know, there is an autocomplete in filament using datalist with limited capability, not advance like yours, it has more features and needed for my project...

ok I look forward into it, even though I need it ASAP, I still can wait...by the way, it's good library...thumbsup for you :)

**Reply by joshhanley** [reply]
@meeftah ah ok makes sense. Thanks for the feedback 🙂

If you want to have a look at the new version, you can look at the `next` branch here https://github.com/joshhanley/livewire-autocomplete/tree/next

Due to a complete API change, I haven't decided yet whether to make this a new major version (v3) or release it as a seperate package which will allow people to slowly migrate to the new syntax. I personally hate nothing worse than to have to change everything when I upgrade a package.

So I need to find a way to make this API update nice and smooth and non-blocking for people.

**Reply by joshhanley** [reply]
@meeftah feel free to start playing with a Filament integration for the new/next version. If you run into any issues or bugs, please let me know and I will try and have a look 🙂

**Reply by meeftah** [reply]
@joshhanley thanks for the heads up... I will start make filament integration 👍
